### PR TITLE
enforce new hash syntax by `rubocop --only HashSyntax --auto-correct`

### PR DESCRIPTION
**Pull request by tagomoris**
I did syntax check for all .rb files by `find . -name '*.rb' | xargs -n 1 ruby -c`

### CODE REVIEW

File: lib/fluent/plugin/filter_record_transformer.rb [hunk 1]
```diff
@@ -60,8 +60,8 @@ def configure(conf)
       end
 
       placeholder_expander_params = {
-        :log           => log,
-        :auto_typecast => @auto_typecast,
+        log: log,
+        auto_typecast: @auto_typecast,
```
Comment (okkez) [inline lib/fluent/plugin/filter_record_transformer.rb]: Breaks align. Is this ok?

File: test/plugin/test_filter_record_transformer.rb [hunk 2]
```diff
@@ -411,35 +411,35 @@ def emit(config, msgs = [''])
...
         es.each_with_index do |(t, r), i|
           actual_results << {
-            :single      => r["single"],
-            :multiple    => r["multiple"],
-            :with_prefix => r["with_prefix"],
-            :with_suffix => r["with_suffix"],
+            single: r["single"],
+            multiple: r["multiple"],
+            with_prefix: r["with_prefix"],
+            with_suffix: r["with_suffix"],
```
Comment (okkez) [inline test/plugin/test_filter_record_transformer.rb]: Breaks align in this file. Is this ok?

File: lib/fluent/plugin/filter_record_transformer.rb (hunk 1 above)
Comment (tagomoris) [inline lib/fluent/plugin/filter_record_transformer.rb]: it has just 2 fields and these are far different from each other. There seems no problem.

File: test/plugin/test_filter_record_transformer.rb (hunk 2 above)
Comment (tagomoris) [inline test/plugin/test_filter_record_transformer.rb]: It seems test code alignments, not to raise any errors.

**Comment by tagomoris** [comment]
@repeatedly I'm about to merge this. Are you ok?

**Comment by repeatedly** [comment]
Dows this syntax work on ruby 1.9?
We sometimes backport master changes to v0.12 and v0.12 should work on ruby 1.9.
I know ruby 1.9 is dead but some users use it on production.

**Comment by tagomoris** [comment]
It's the new feature for Ruby 1.9.
http://effectif.com/ruby/update-your-project-for-ruby-19-hash-syntax

**Comment by repeatedly** [comment]
I see. Okay.

**Comment by tagomoris** [comment]
Merged.

**Pull request merged**